stock_warehouse_analytic_purchase/models/purchase_order_line.py

```diff
@@ -0,0 +1,16 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import _, models, api
```

Comment:
Remove unused import:
```suggestion
from odoo import models, api
```